Fix VK OAuth not working

PickRelated opened the pull request:
Bumped VK provider API version. Prior to this VK OAuth used to fail silently without creating a user. As it turned out - VK API responded with `versions prior to 5.21 are not supported`.

petersg83 commented:
Thanks for your contribution!
I think it can be merged but I have a few questions:
- Does it work for you ? I tried it but I got an error because the user email is not given in vk response.
- If it works for you, what have you done to make it work?

Thanks !

PickRelated commented:
> * Does it work for you ? I tried it but I got another error because the user email is not given in vk response.
> * If it works for you, what have you done to make it work?

You get your email on the frontend and pass it to backend in exchange for token. Something like this (on frontend):

```js
    const params = location.search
      .replace('?', '')
      .split('&')
      .map(e => e.split('='))
      .reduce((acc, [key, value]) => {
        acc[decodeURIComponent(key)] = decodeURIComponent(value)
        return acc
      }, {})

    axios.get('/auth/vk/callback', params)
```
 Not very convenient, but it has nothing to do with my PR. :)

petersg83 commented:
Ok thank you, it wasn't working on my side because my vk account actually doesn't have an email. So it's good for me :)
Thanks !
